src/pages/Home/homepage.jsx

```diff
@@ -1,81 +1,106 @@
+           </div>
+            <p>Titan</p>
+           <button className="home-btn">Add To Cart</button>
+           </div>
+           <div className="card">
+              <div className="card-img-box">
+           <img src= "https://assets.myntassets.com/f_webp,dpr_1.5,q_60,w_210,c_limit,fl_progressive/assets/images/17161252/2022/2/15/535b9721-0249-4f06-8054-7adfdc6462131644916584800EmporioArmaniMenBlackSkeletonDialBlackStainlessSteelStrapsAn1.jpg" alt="" />
+           </div>
+            <p>Titan</p>
+           <button className="home-btn">Add To Cart</button>
+           </div>
+           <div className="card">
+              <div className="card-img-box">
+           <img src= "https://assets.myntassets.com/f_webp,dpr_1.5,q_60,w_210,c_limit,fl_progressive/assets/images/17161230/2022/2/15/5e452a7f-fb07-4080-acec-5ae580d4478a1644916949644EmporioArmaniMenBlueDialSilverTonedStainlessSteelBraceletSty1.jpg" alt="" />
+           </div>
+            <p>Titan</p>
+           <button className="home-btn">Add To Cart</button>
+           </div>
+           <div className="card">
+              <div className="card-img-box">
+           <img src= "https://assets.myntassets.com/f_webp,dpr_1.5,q_60,w_210,c_limit,fl_progressive/assets/images/15471534/2021/9/20/6e940c37-6fe0-43ab-8ffd-10b1918bb0b31632137547618-Fastrack-Unisex-Watches-641632137547214-1.jpg" alt="" />
+           </div>
+            <p>Titan</p>
+           <button className="home-btn">Add To Cart</button>
+           </div>
+                    
             </div>
+            <button className="banner-button">Browse Now</button>
 
+          </div>
+        </div>
             {/* -----------------------------------SLIDING */}
 
-            <h2>Our Collections</h2>
-            <div className="collection-links">
-                <p className="link-men" onClick = {() => setDisplay("men")}>Men</p>
-                <p className="link-women" onClick = {() => setDisplay("women")}>Women</p>
-                <p className="link-luxury" onClick = {() => setDisplay("luxury")}>Luxury</p>
-            </div>
-            {display === "men" && <MenDisplay/>}
-            {display === "women" && <WomenDisplay/>}
-            {display === "luxury" && <LuxuryDisplay/>}
 
          {/* ---------------------------BRANDS------------------------------ */}
-
-    <h2>Brands</h2>
+         <div className="home-category-banner col-12">
+           <img src={home1} alt="" />
+         </div>
+         <div className="home-category-banner2 col-12">
+           <img src={home2} alt="" />
+         </div>
+         {/* ---------------------------------- */}
+         <h2>Brands</h2>
      <div className="brand-container">
          <img src= {brand1} alt="" />
          <img src={brand2} alt="" />
          <img src={brand3} alt="" />
          <img src={brand4} alt="" />
          <img src={brand5} alt="" />
      </div>
+     {/* -------------------------- */}
```

Comment:
I don't this it's necessary to used this commented separation in your jsx file, usual practice is to use it in css.